samples/python/greedy_causal_lm/README.md

```diff
@@ -22,6 +26,18 @@ Discrete GPUs (dGPUs) usually provide better performance compared to CPUs. It is
 
 See https://github.com/openvinotoolkit/openvino.genai/blob/master/src/README.md#supported-models for the list of supported models.
 
+## Run with optional LoRA adapters
+
+LoRA adapters can be connected to the pipeline and modify generated text. Adapters are supported in Safetensors format and can be downloaded from public sources like [Civitai](https://civitai.com) or [HuggingFace](https://huggingface.co/models) or trained by the user. Adapters compatible with a base model should be used only. A weighted blend of multiple adapters can be applied by specifying multiple adapter files with corresponding alpha parameters in command line. Check `lora.py` source code to learn how to enable adapters and specify them in each `generate` call.
+
+Here is an example how to run the sample with a single adapter. First download adapter file from TODO page manually and save it as TODO. Or download it from command line:
+
+#TODO command to download adapter
+
+Then run `lora.py`:
+
+#TODO command to run lora.py with adapter
```

Comment:
Why is it postponed?

Reply:
@slyalin asked to postpone it, as this readme needs to be rewritten, with more suitable model. And we are not sure which model to use yet.

Reply:
There's already a comment
> // not supported properties are not checked, as these properties are passed to compile(), which will throw exception in case of unsupported property

so that should be fine